---
permalink: /research/
title: "Research"
author_profile: true
redirect_from: 
  - /research/
  - /research.html
---

<img src="/images/jasper.png" 
        alt="Jasper_Ridge" 
        width="900" 
        height="200" 
        style="display: block; margin: 0 auto" />

The Wild Side of <i>E. coli</i>
------
The Anthropocene represents a period of rapid environmental change, affecting not only individual species, but also the interactions and communities of which they are a part, including the microscopic communities that exist within individual hosts. Encroachment of humans into wildlands introduces human-associated bacteria into previously na√Øve environmental and wild animal gut bacterial communities. The extent of the wildland-urban interface has dramatically increased in the past few centuries and continues to expand today, thereby increasing the rate of microbial spillover and spillback between wild animal and human populations. Environmental pollution by human and domestic animal waste, antibiotics, agrochemicals, and more, also impacts microbial communities, which begets the question: Are wild animal guts truly "wild" anymore?

<img src="/images/onehealth.png" 
        alt="One Health microbial transmission" 
        width="400" 
        height="400" 
        style="display: block; margin: 0 auto" />

For my dissertation research, I sought to better understand the propensity of wild communities to serve as reservoirs and "melting pots" of infectious and drug-resistant diseases that could severely impact global public health, but also to understand the extent to which wildlife spread such diseases versus the extent of "spillback" from humans to wildlife. To do this, I enlisted the help of a common and primarily commensal resident of the vertebrate gut, <i>Escherichia coli</i>. Considering how much we know about <i>E. coli</i> in humans and domestic animals, it remains strikingly unexamined in wild animals, despite the fact that they are frequently blamed for pathogenic <i>E. coli</i> contamination events in agricultural production. 

<img src="/images/map.png" 
        alt="Geographic gaps in research" 
        width="700" 
        height="500" 
        style="display: block; margin: 0 auto" />

**Figure 1.** (a) Geographic gaps in research on <i>E. coli</i> in wild animals. Increasing gradient from pale blue, denoting a study count of 1, to dark blue, denoting a count of 10 (USA) or 11 (Portugal). No studies fitting our inclusion criteria were found in countries in grey. (b) Heat map depicting the number of wild animal host species investigated in each country. The colour gradient illustrates increasing number of species per country from yellow to dark orange. Min (1) and max (166) are labelled (from Lagerstrom and Hadly, 2021).
{: .notice}

I began by conducting a comprehensive [literature review](kate-lagerstrom.github.io/files/Lagerstrom_2021.pdf) of what we knew about <i>E. coli</i> in wild animals globally to-date and identified important knowledge gaps in the geographic range and host species diversity investigated so far (fig. 1), as well as shortcomings in the more popular sampling and statistical methods used to conduct such investigations. I wondered then, if more comprehensive sampling were conducted across a broader range of hosts and from more individuals within a host species, could we identify which factors contribute to the composition of <i>E. coli</i> in a wild animal? In figure 2, I describe a number of possible determinants based in ecological theory, some of which have been previously suggested. 

<img src="/images/tree.png" 
        alt="flow chart" 
        width="700" 
        height="500" 
        style="display: block; margin: 0 auto" />
        
**Figure 2.** Potential determinants of <i>E. coli</i> phylogroup composition. A flow diagram outlining potential factors influential to determining the community structure of the phylogroups within a host related to the environment, the host, and <i>E. coli</i> itself (from Lagerstrom and Hadly, 2023).
{: .notice}

To address this question, I surveyed 163 scat samples from 17 different host species residing at [Jasper Ridge Biological Preserve](https://jrbp.stanford.edu/) (JRBP) CA, USA and obtained a total of nearly 1,800 <i>E. coli</i> isolates. I assessed the diversity and distribution of <i>E. coli</i> among this wild animal community using a low-cost and rapid multiplex PCR method known as the [Clermont Method](https://enviromicro-journals.onlinelibrary.wiley.com/doi/10.1111/1758-2229.12019). The results demonstrated that diversity recovered at the phylogroup level was strongly linked to sampling depth within and between hosts. These findings differentiated themselves from previous studies that reached debatable conclusions because they were based on inadequate sample sizes. Indeed, many of the previously identified trends in phylogroup distribution and host characteristics were no longer sound when deeper sampling was conducted. Thus, I employed a common quantitative ecological sampling method based on calculating saturation curves for phylogroup discovery within each wild host species individually (fig. 3). In doing so, I demonstrated a practical approach for developing ecologically informed, measurable, and verifiable standards to assess the true diversity of <i>E. coli</i> in wild animal guts that will also contribute to our understanding of their potential to act as reservoirs of human pathogens. You can read the full research article [here](kate-lagerstrom.github.io/files/Lagerstrom_2023.pdf). Though inexpensive and effective at distinguishing sub-species level diversity, the <i>E. coli</i> phylogeny remains diverse enough such that even this level of investigation may be insufficient for accurately predicting the risk of zoonotic spillover or rate of development and spread of genes conferring antimicrobial resistance (AMR) through wild animal networks and ecosystems.

<img src="/images/saturation.png" 
        alt="flow chart" 
        width="500" 
        height="900" 
        style="display: block; margin: 0 auto" />

**Figure 3.** Sampling host-associated <i>E. coli</i> to phylogroup-level saturation. Rarefaction curves for the number of <i>E. coli</i> colonies necessarily sampled to reach phylogroup-level saturation for each host species. Not shown are species with only 1 phylogroup present (n = 1, including Lagomorpha, dusky-footed woodrat, long-tailed weasel, and western fence lizard). Dotted blue lines: saturation curve for the richest scat sample of a host species. Dotted black lines: saturation curve for the least rich individual. Yellow outlines indicate the domestic host species. Asterisks denote multi-individual samples (from Lagerstrom and Hadly, 2023).
{: .notice}

Therefore, I selected a representative subset of the diversity acquired from each host species for whole genome sequencing (WGS) and subsequent high-resolution investigations of the rate of carriage and diversity of AMR genes, virulence factors associated with pathogenic <i>E. coli</i>, and genomic diversity across JRBP (fig. 4). In 143 <i>E. coli</i> isolates, vast genetic diversity was captured that belonged to all 8 major phylogroups (and 2 cryptic clades), and 89 unique sequence types (STs). In this small preserve, we found the global phylogroup diversity of <i>E. coli</i>, high pathogenic load, and AMR in wild animals. Many of the STs (~20%) were strongly human-associated and 17% carried one or more AMR gene. Additionally, 46% of these <i>E. coli</i> isolates qualified as pathogenic (to humans) based on the repertoire of virulence factors they carried. You can read the full research article [here](kate-lagerstrom.github.io/files/Lagerstrom_2024.pdf). These results support a role for wild communities in harboring and potentially spreading microbes of concern to human health, but also suggest that humans contribute <i>E. coli</i> to the wildlife microbiome. These findings add to our understanding of host-microbe evolution and provide insight into community-level transmission of <i>E. coli</i>.  

<img src="/images/phylogeny.png" 
        alt="phylogenetic tree" 
        width="700" 
        height="700" 
        style="display: block; margin: 0 auto" />
        
**Figure 4.** Genetic and ecological structure of <i>E. coli</i> from JRBP
RECOPHY phylogeny of 143 de novo constructed <i>E. coli</i> genomes from animals at JRBP and including reference genomes MG1655 and O157:H7, rooted by <i>E. fergusonii</i>. Colored strips from inside out indicate host status, <i>E. coli</i> phylogroup, AMR, and pathotype (some isolates met criteria for multiple pathotype assignments). Graphic created in iTOL v5 (from Lagerstrom, <i>et. al.</i>, 2024).
{: .notice}

Through more investigations like these, my motivating objective is to understand the relationships between humans, wildlife, and our microbial partners and pathogens, to better serve both wildlife and the public through enhancing our knowledge about the ecology and evolution of bacteria in the wild.

<img src="/images/gotecoli.png" 
        alt="Poop emoji drawn from E. coli" 
        width="300" 
        height="300" 
        style="display: block; margin: 0 auto" />
